openlibrary/plugins/importapi/code.py

```diff
@@ -128,17 +135,17 @@ def POST(self):
         except ClientException as e:
             return self.error('bad-request', **json.loads(e.json))
 
-    def reject_non_book_marc(self, marc_record, **kwargs):
-        details = 'Item rejected'
-        # Is the item a serial instead of a book?
-        marc_leaders = marc_record.leader()
-        if marc_leaders[7] == 's':
-            return self.error('item-is-serial', details, **kwargs)
+def raise_non_book_marc(marc_record, **kwargs):
```

Comment:
Is there a case when it would not raise? The name seems to be in the same vein as requests' `raise_for_http_status`, which raises if it finds an error http status, so the name looks ok (unless you mean there are cases when it doesn't raise that it should?).

Reply:
I agree with @cdrini here, resolving